examples/logs/exporters/otlp_grpc.php

```diff
@@ -30,19 +29,18 @@
         (new LogRecordLimitsBuilder())->build()->getAttributeFactory()
     )
 );
-$logger = $loggerProvider->getLogger('demo', '1.0', 'http://schema.url', ['foo' => 'bar']);
-$eventLogger = new EventLogger($logger, 'my-domain');
+$eventLoggerProvider = new EventLoggerProvider($loggerProvider);
+$eventLogger = $eventLoggerProvider->getEventLogger('demo', '1.0', 'http://schema.url', ['foo' => 'bar']);
 
-$eventLogger->logEvent(
-    'foo',
-    (new LogRecord(['foo' => 'bar', 'baz' => 'bat', 'msg' => 'hello world']))
-        ->setSeverityText('INFO')
-        ->setSeverityNumber(SeverityNumber::SEVERITY_NUMBER_INFO)
+$eventLogger->emit(
+    name: 'foo',
+    payload: ['foo' => 'bar', 'baz' => 'bat', 'msg' => 'hello world'],
+    severityNumber: Severity::INFO
 );
 
-$eventLogger->logEvent(
+$eventLogger->emit(
     'foo',
```

Comment:
```suggestion
    'bar',
```

 `foo` is used on L36 with different payload schema.
> All events with the same `name` have `Payloads` that conform to the same schema